include/odp/api/spec/classification.h

```diff
@@ -383,8 +383,28 @@ typedef struct odp_red_param_t {
 } odp_red_param_t;
 
 /** Back pressure (BP)
- * When back pressure is enabled for a particular flow, the HW can send
- * back pressure information to the remote peer indicating a network congestion.
+ *
+ * When back pressure is enabled, HW can send back pressure to the remote peer
+ * indicating resource shortage. Depending on the implementation, back pressure
+ * can be asserted when the fill level of the destination queue of a CoS
+ * exceeds the configured threshold or when the utilization of the packet pool
+ * associated with a CoS exceeds the configured threshold.
+ *
+ * If multiple CoS'es configure back pressure for the same pktio, pool/queue
+ * and pfc level, then back pressure is sent as long as any of the CoS'es
+ * assert back pressure. There does not have to be packets flowing through
+ * a CoS node for the CoS to assert back pressure, because packets may be
+ * directed to the same queue/pool by some other CoS node (which might not
+ * even have backpressure configured).
```

Comment:
Should we generalize this, so that also any other ODP component may utilize the same queue/pool?

Reply:
Mentioned also other ODP components and the ODP application in v2.